### ISSUE TITLE
V1.7.2 is showing values for the wrong fields

### ISSUE THREAD

**Issue by mfairleigh** [MiniKeePass/MiniKeePass#669]
For example, for one of my cars, for VIN number it shows a password for a website which is also in the same vehicle record.  The VIN number value isn’t shown anywhere.  The values are correct in desktop Keepass.  I’ve tried re-downloading the file with the same results.  iOS current version.

**Comment by mfairleigh** [comment]
Is this app still being maintained?  This is a fairly serious bug - and one that cropped up before and was fixed once.

**Comment by tflo** [comment]
> Is this app still being maintained?

I don’t think so.

Some time ago I put together some alternatives:

- [**Strongbox**](https://strongboxsafe.com)
	- Way better than MiniKeePass ever was (GUI, options, features)
	- [Seems to be](https://github.com/mmcguill/Strongbox) open sourced, at least partially
	- Free version available, but the full version is moderatly expensive: €29 for a lifetime purchase, and some subscription options like for example €10/year.
	- It can – optionally – edit the iCloud-served kdbx file in-place (which means live sync)
	- Can create Diceware passphrases, and you can select the dictionary out of an ample list of acknowledged Diceware dictionaries (including localized ones!)
	- Mac version available
‌
- [**KeePassium**](https://keepassium.com)
	- Way better than MiniKeePass ever was (GUI, options, features)
	- Not open source
	- Free version available, but no one-time purchase available. Only subscriptions like for example €12/year.
	- It does edit – non-optionally – the iCloud-served kdbx file in-place (which means live sync)
‌
- [**Keepass Touch**](https://www.innervate.de/keepass-touch.html)
	- Seems comparable to MiniKeePass, but I didn’t really test it, since in its current version [as of 2019-08-01] it refuses to open database files from iCloud.
	- Not open source
	- Free version available, and the full version is not expensive: €1

All three are actively developed; Strongbox and KeePassium very actively (updated some days ago).

Currently I’m using/evaluating Strongbox, and I’m pretty content with it. 

– Tom

**Comment by mfairleigh** [comment]
Yes, I saw your post right after I posted that.  I’m currently evaluating strongbox.  I think the subscription implementation is silly, but time will tell.  Thanks!MikeOn September 27, 2019 at 5:01 AM Tom <notifications@github.com> wrote:  Is this app still being maintained?I don’t think so.Some time ago I put together some alternatives:StrongboxWay better than MiniKeePass ever was (GUI, options, features)Seems to be open sourced, at least partiallyFree version available, but the full version is moderatly expensive: €29 for a lifetime purchase, and some subscription options like for example €10/year.It can – optionally – edit the iCloud-served kdbx file in-place (which means live sync)Can create Diceware passphrases, and you can select the dictionary out of an ample list of acknowledged Diceware dictionaries (including localized ones!)Mac version availableKeePassiumWay better than MiniKeePass ever was (GUI, options, features)Not open sourceFree version available, but no one-time purchase available. Only subscriptions like for example €12/year.It does edit – non-optionally – the iCloud-served kdbx file in-place (which means live sync)Keepass TouchSeems comparable to MiniKeePass, but I didn’t really test it, since in its current version [as of 2019-08-01] it refuses to open database files from iCloud.Not open sourceFree version available, and the full version is not expensive: €1All three are actively developed; Strongbox and KeePassium very actively (updated some days ago).Currently I’m using/evaluating Strongbox, and I’m pretty content with it.– Tom—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
